src/main/cpp/t05_power.cpp

```diff
@@ -28,8 +28,18 @@
 using namespace std;
 
 //function power
+int power(int a, int n)
+{
+	int s = 1;
+	if (n > 0) { s = s*a*power(a, n - 1); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 's' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4519357)